packages/foreman/tfm/tfm.spec

```diff
@@ -137,6 +143,8 @@ Requires: %{scl_runtime}
 Requires: %{scl_runtime}-assets
 Requires: %{scl_prefix_ror}scldevel
 Requires: %{scl_prefix_ruby}scldevel
+Requires: %{scl_prefix_nodejs}scldevel
+Requires: %{scl_prefix_nodejs}build
```

Comment:
There are certain macros that need to be available at build time for nodejs packages, e.g. nodejs_sitelib